clients/avro/pom.xml

```diff
@@ -116,8 +118,9 @@ http://maven.apache.org/xsd/maven-4.0.0.xsd">
           <version>${confluent.version}</version>
           <configuration>
               <schemaRegistryUrls>
-                  <param>http://localhost:8081</param>
+                  <param>${schemaRegistryUrl}</param>
               </schemaRegistryUrls>
+              <userInfoConfig>${schemaRegistryBasicAuthUserInfo}</userInfoConfig>
               <subjects>
                   <transactions-value>src/main/resources/avro/io/confluent/examples/clients/basicavro/Payment2a.avsc</transactions-value>
               </subjects>
```

Comment:
What do you think about adding in the `schemaTypes` section here (https://docs.confluent.io/current/schema-registry/develop/maven-plugin.html#schema-registry-test-compatibility) now that there is support for PB & JSON?  Even though the default is AVRO, a reader will be able see how to change their's to PROTOBUF if that's the type they use (the associated JIRA talks about the other schema types as well):

```
<schemaTypes>
  <transactions-value>AVRO</transactions-value>
</schemaTypes>
```

Reply:
@rspurgeon nice suggestion, done.